libs/keyboard/keyboard.ts

```diff
@@ -156,117 +156,6 @@ const enum KeyboardModifierKey {
  */
 //% icon="\uf11c" color="#303030"
 namespace keyboard {
-    // prevent catastrophic pounding of keyboard
```

Comment:
this is no longer needed?

Reply:
The bug was c++ was emitting down instead of up

Reply:
Otherwise it’s already updated